README.md

```diff
@@ -1,24 +1,75 @@
-# README
+# Project Name
+Transaction app 
+The Transaction app will be a classic example of a Transaction website. We created a fully functional website that will show the list of groups and Transaction and empower user to interact with them 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
 
-Things you may want to cover:
+* Ruby on Rails
+* Rspec
+* Psql
 
-* Ruby version
+## Video
+This is the link to introductory Video: [Video](https://www.loom.com/share/a315c9a60e4249038aeb5b972461e529)
 
-* System dependencies
 
-* Configuration
+## Live Link
+This is the link to heroku link: [Link](https://vic-transaction.herokuapp.com/)
```

Comment:
As required, The deployed project should work as expected, Currently, I am getting an error with your deployed link.

![image](https://user-images.githubusercontent.com/66112760/154801060-14e6d3a2-a714-4cd3-b038-e2692dbf9252.png)

Kindly fix this.

Reply:
- The link is working here and I have sent to other people and its working on their side too
![image](https://user-images.githubusercontent.com/66097371/154801557-d31726a0-bbb4-40dc-bb24-6589e51cd8d4.png)

Reply:
- I was referring to the sign-up route, I was unable to signup, Glad you have resolve the bug